cmd/containerd-shim-runhcs-v1/task_wcow_podsandbox.go

```diff
@@ -236,6 +239,32 @@ func (wpst *wcowPodSandboxTask) DumpGuestStacks(ctx context.Context) string {
 	return ""
 }
 
+func (wpst *wcowPodSandboxTask) Share(ctx context.Context, req *shimdiag.ShareRequest) error {
+	if wpst.host == nil {
+		return errors.New("task is not isolated")
+	}
+	guestReq := guestrequest.GuestRequest{
+		ResourceType: guestrequest.ResourceTypeMappedDirectory,
+		RequestType:  requesttype.Add,
+	}
+	options := &hcsschema.VirtualSmbShareOptions{
+		NoDirectmap: wpst.host.DevicesPhysicallyBacked(),
+	}
+	if req.ReadOnly {
+		options.ReadOnly = true
+		options.CacheIo = true
+		options.ShareRead = true
+		options.ForceLevelIIOplocks = true
+	}
+	guestReq.Settings = &hcsschema.MappedDirectory{
+		ContainerPath: req.UvmPath,
+		HostPathType:  "VirtualSmbShareName",
```

Comment:
Where did this value come from? Can we not hardcode it?

Reply:
It's not in our generated schema files but I can add it. Found this from hcsdiag

Reply:
I would not put it in a generated schema file. I think we should try to avoid that where possible so we don't accidentally overwrite values we need later.

Reply:
I worded that poorly, it IS apart of the HCS schema but not apart of our go file schema generation. (as seems to be the case for everything we need nowadays -___-)